dist/adaptive-html.cjs.js

```diff
@@ -93,12 +106,18 @@ function createCard(elements) {
     type: cardTypes.adaptiveCard,
     body: [],
     actions: [],
-    version: '1.0'
+    version: '1.2'
   };
-  var body = toArray(elements);
+  var body = toArray(elements); // remove extra container wrapping for a single element that only has required properties (type, items)
+  // otherwise leave the container wrap since there could be additional styling
+  // https://adaptivecards.io/explorer/Container.html
+
+  if (Array.isArray(elements) && elements.length === 1) {
+    var singleElement = elements[0];
 
-  if (Array.isArray(elements) && elements.length === 1 && isContainer(elements[0])) {
-    body = toArray(unwrap(elements[0]));
+    if (isContainerWithRequiredProperties(singleElement)) {
+      body = toArray(unwrap(singleElement));
```

Comment:
is `singleElement.items` ever not an array at this point? Could this line just be: 

```
unwrap(singleElement)
```